README.rst

```diff
@@ -302,10 +302,10 @@ The BPF code templates are dual-licensed under the
 and the `2-Clause BSD License <bsd-license_>`__
 (you can use the terms of either license, at your option).
 
-.. _`Why Cilium?`: http://docs.cilium.io/en/stable/intro/#why-cilium
-.. _`Getting Started`: http://docs.cilium.io/en/stable/gettingstarted/
-.. _`Architecture and Concepts`: http://docs.cilium.io/en/stable/concepts/
-.. _`Installing Cilium`: http://docs.cilium.io/en/stable/gettingstarted/#installation
+.. _`Why Cilium?`: https://docs.cilium.io/en/stable/overview/intro
+.. _`Getting Started`: https://docs.cilium.io/en/stable/#getting-started
+.. _`Architecture and Concepts`: https://docs.cilium.io/en/stable/overview/component-overview/
+.. _`Installing Cilium`: https://docs.cilium.io/en/stable/gettingstarted/k8s-install-default/
```

Comment:
There are a few more `http` links in this block, this is maybe an occasion to switch them to `https` as well.

Looks all good otherwise.

Reply:
good point, done